extend/common-interface/folders.md

```diff
@@ -152,20 +152,20 @@ will contain only manifest files, extended with an additional
 
 **Note**: Exchanging data via ABS is currently only available for input mapping.
 
-## Exchanging Data via Workspace
+## Exchanging Data via Database Workspace
```

Comment:
ok, it's probably worth saying here that this applies to only Storage Tables, and Storage Files still go to the local mapping

the  we can say in the # Exchanging Data via File System Workspace that it can be used by both files and tables